embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/config/LocalConfigParser.kt

```diff
@@ -114,21 +115,13 @@ internal object LocalConfigParser {
                 " embrace.disableMappingFileUpload=true to gradle.properties."
         }
 
-        val enabledStr = when {
-            ndkEnabled -> "enabled"
-            else -> "disabled"
-        }
-        logger.logInfo("Native crash capture is $enabledStr")
         var configs: SdkLocalConfig? = null
         if (!sdkConfigs.isNullOrEmpty()) {
             Systrace.traceSynchronous("deserialize-sdk-config") {
                 try {
                     configs = serializer.fromJson(sdkConfigs, SdkLocalConfig::class.java)
                 } catch (ex: Exception) {
-                    logger.logError(
-                        "Failed to parse Embrace config from config json file.",
-                        ex
-                    )
+                    logger.trackInternalError(InternalErrorType.CONFIG_DESERIALIZATION_FAIL, ex)
```

Comment:
Failing deserialization can point to a bug in the config service setup, so lets log an error just in case.